packages/edit-post/src/index.js

```diff
@@ -147,6 +147,9 @@ export function initializeEditor(
 	);
 }
 
+// esm-example
+import( '@wordpress/esm-example' ).then( ( mod ) => mod.run() );
```

Comment:
Is the following case in webpack plugin necessary for the import above?

```js
if ( ESM_PACKAGES.includes( request ) ) {
 	return 'import ' + request;
}
```

Reply:
yes, this config ensures webpack keeps the import as is and do not try to bundle or resolve the `@wordpress/esm-example` dependency. 

Ideally, I'd like webpack to rewrite it `importShim( '@wordpress/esm-example' ).then( ( mod ) => mod.run() );` but I was not able to do so.